sip-ri/src/test/java/gov/nist/javax/sip/parser/extensions/TargetDialogParserTest.java

```diff
@@ -8,17 +8,16 @@ public class TargetDialogParserTest extends ParserTestCase {
 
     @Override
     public void testParser() {
-        String[] targetDialogHeaders = {
+        String[] to = {
                 "Target-dialog: f0b40bcc-3485-49e7-ad1a-f1dfad2e39c9@10.5.0.53\n",
                 "Target-dialog: f81d4fae-7dec-11d0-a765-00a0c91e6bf6@foo.bar.com\n",
                 "Target-dialog: kl24ahsd546folnyt2vbak9sad98u23naodiunzds09a3bqw0sdfbsk34poouymnae0043nsed09mfkvc74bd0cuwnms05dknw87hjpobd76f\n",
-                "Target-dialog: 12345th5z8z;local-tag=tozght6-45;remote-tag=fromzght789-337-2\n"
+                "Target-dialog: 12345th5z8z;local-tag=localzght6-45;remote-tag=remotezght789-337-2\n"
```

Comment:
I would test this one separately out of the for loop below in its own dedicated test and do assertTrue(targetDialog.getCallId(), "12345th5z8z") and
assertTrue(targetDialog.getLocalTag(), "localzght6-45") and assertTrue(targetDialog.getRemoteTag(), "remotezght789-337-2") and assertTrue(targetDialog.encodeBody().toString(), "Target-dialog: 12345th5z8z;local-tag=localzght6-45;remote-tag=remotezght789-337-2") to ensure that if the implementation is modified we can catch any regressions on parsing callId and parameters. I think this would have caught the issue of the previous incorrect implementation

Reply:
![Screenshot from 2024-03-26 13-26-51](https://github.com/mobius-software-ltd/corsac-sip/assets/118014254/8917eb1a-5843-49f3-902b-8e14c0c8d615)
modified, here is the output, tests all passed